src/gpio/mod.rs

```diff
@@ -331,5 +333,40 @@ macro_rules! define_led {
     }
 }
 
+/// Opaque AFRL register
+pub struct AFRL<GPIO>(PhantomData<GPIO>);
+/// Opaque AFRH register
+pub struct AFRH<GPIO>(PhantomData<GPIO>);
+/// Opaque MODER register
+pub struct MODER<GPIO>(PhantomData<GPIO>);
+/// Opaque OTYPER register
+pub struct OTYPER<GPIO>(PhantomData<GPIO>);
+/// Opaque PUPDR register
+pub struct PUPDR<GPIO>(PhantomData<GPIO>);
+
```

Comment:
Please add comment that these are only common pins among all stm32l4 boards for future reference